### ISSUE TITLE
How to get target image embeddings for retrieval with AlphaCLIP?

### ISSUE THREAD

**Issue by raghav-akridata** [SunzeY/AlphaCLIP#59]
Hey!
Is it mandatory to supply the alpha mask for embedding target images? Masking the entire target image with `np.ones` does not result in a good accuracy for me. Am I missing something?

```python
# prep a white mask
binary_mask = np.ones((336, 336), dtype=np.uint8)
alpha = mask_transform((binary_mask * 255))
alpha = alpha.cuda().unsqueeze(dim=0)

# extract features
with torch.no_grad():
    target_image_features = model.encode_image(target_image, alpha)
```

**Comment by SunzeY** [comment]
check out our demo code, 1 is not the positive threshold for our model.
